install

```diff
@@ -308,7 +308,7 @@ function createDockerComposeFile() {
     if [[ -n "$licenseAccessKey" ]]; then
       echo "LICENSE_ACCESS_KEY=${licenseAccessKey}" >> $dockerEnvFile
     fi
-    if [[ -n "$licenseOfflineMode" ]]; then
+    if [[ $licenseOfflineMode = true ]]; then
```

Comment:
make line 48
licenseOfflineMode=false
and line 92
--licenseofflinemode) licenseOfflineMode=true;;